client/src/widgets/PhotoGallery.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <swiper-container
+    :slides-per-view="1"
+    :space-between="15"
+    class="photoGallery"
+  >
+    <swiper-slide v-for="photo in photos" :key="photo"
+      ><div
+        :style="`background-image: url(${photo})`"
+        class="photoGallery__photo"
+      ></div
+    ></swiper-slide>
+  </swiper-container>
+</template>
+
+<script setup lang="ts">
+import { register } from 'swiper/element/bundle';
+
+register();
+
```

Comment:
Trzeba pamietac zeby potem usunac testowy foty jak serwis sie podlaczy a co do tego register to trzeba go wywalic gdzie indziej najlepiej do main.ts bo to trzeba wywolac tylkjo raz i custom element od nich jest dostepny globalnie
![image](https://github.com/AdamTroszczynski/ViewHunters/assets/34286232/2b4edfb5-f7e2-4ed0-8a47-151e133d07c7)

Reply:
Ok już poprawiam